src/main/java/duke/Command.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+/**
+ * Encapsulates an Abstract Command class
+ */
+public abstract class Command {
+    private String[] parsedCommand;
```

Comment:
Maybe this can be made protected as there are classes that extend from it so they can directly access this variable instead of using a getter.

Reply:
Yes very good idea thank you.